python/ray/tune/integration/ray_train.py

```diff
@@ -13,6 +14,12 @@
 class TuneReportCallback(UserCallback):
     """Propagate metrics and checkpoint paths from Ray Train workers to Ray Tune."""
 
+    def __init__(self):
+        assert _in_tune_session(), "TuneReportCallback must be used in a Tune session."
+        self.training_actor_item_queue = (
```

Comment:
nit: make it private 
```suggestion
        self._training_actor_item_queue = (
```